packages/gatsby/src/utils/__tests__/get-latest-gatsby-files.ts

```diff
@@ -32,29 +81,53 @@ const getMockAPIFile = (): IAPIResponse => {
 }
 
 describe(`default behavior: has network connectivity`, () => {
-  beforeEach(() => {
-    fs.pathExists.mockResolvedValueOnce(false)
-    axios.get.mockResolvedValueOnce({ data: getMockAPIFile() })
-  })
+  describe(`getLatestAPIs`, () => {
+    beforeEach(() => {
+      axios.get.mockResolvedValueOnce({ data: getMockAPIFile() })
+    })
 
-  it(`makes a request to unpkg to request file`, async () => {
-    const data = await getLatestAPIs()
+    it(`makes a request to unpkg to request file`, async () => {
+      const data = await getLatestAPIs()
 
-    expect(axios.get).toHaveBeenCalledWith(
-      expect.stringContaining(`unpkg.com`),
-      expect.any(Object)
-    )
-    expect(data).toEqual(getMockAPIFile())
+      expect(axios.get).toHaveBeenCalledWith(
+        expect.stringContaining(`unpkg.com`),
+        expect.any(Object)
+      )
+      expect(data).toEqual(getMockAPIFile())
+    })
+
+    it(`writes apis.json file`, async () => {
+      const data = await getLatestAPIs()
+
+      expect(fs.writeFile).toHaveBeenCalledWith(
+        expect.stringContaining(`latest-apis.json`),
+        JSON.stringify(data, null, 2),
+        expect.any(String)
+      )
+    })
   })
 
-  it(`writes apis.json file`, async () => {
-    const data = await getLatestAPIs()
+  describe(`getLatestAdapters`, () => {
+    beforeEach(() => {
+      axios.get.mockResolvedValueOnce({ data: latestAdaptersMarker })
+    })
 
-    expect(fs.writeFile).toHaveBeenCalledWith(
-      expect.stringContaining(`latest-apis.json`),
-      JSON.stringify(data, null, 2),
-      expect.any(String)
-    )
+    it.only(`loads .js modules`, async () => {
```

Comment:
Remove `.only`?

Reply:
🤦 

dropped in https://github.com/gatsbyjs/gatsby/pull/38475/commits/9ef2b7e3daf7fbe33acaf74582b8a3dc36231007